tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -107,7 +107,7 @@
       // The endpoint that serves trace viewer app.
       traceViewerBaseUrl: {
         type: String,
-        value: () => getRouter().pluginRoute("profile", "/trace_viewer"),
+        value: "/trace_viewer_index.html",
```

Comment:
Is this change really correct? Its use in `_maybeUpdateTool` (viz., `this.traceViewerBaseUrl + "?trace_data_url=" + …`) makes me skeptical. If so, could you elaborate?